lib/attributor/attribute.rb

```diff
@@ -137,8 +137,7 @@ def example_from_options(parent, context)
       when ::Regexp
         val.gen
       when ::Array
-        # TODO: handle arrays of non native types, i.e. arrays of regexps.... ?
-        val.pick
+        val
```

Comment:
It works as written, for everything but collections, though. An array of examples means to pick one of them.